test/unit/helpers/document_helper_test.rb

```diff
@@ -81,7 +81,7 @@ class DocumentHelperTest < ActionView::TestCase
     controller.stubs(:url_options).returns(
       action: "show",
       controller: "world_locations",
-      locale: "it",
+      locale: "de",
```

Comment:
Not major but how come this changed?

Reply:
Tests were seemingly failing, further down you can see it's testing for a German locale but it's passing Italian

Reply:
How weird! Feel free to resolve my dude.